docs/configuration/index.rst

```diff
@@ -20,4 +20,5 @@ The following structure respresent the cli structure.
    trafficpolicy/index
    vpn/index
    vrf/index
-   zonepolicy/index
\ No newline at end of file
+   zonepolicy/index
+   container/index
```

Comment:
Can you please correct the place of in an alphabetic order. So please put in on top. Thanks

Reply:
Got it, just moved that to the top